Тесты готовы. Аллюр готов. PageObject'ы на три разных страницы готовый.

apologet-insane opened the pull request.

apologet-insane committed:
Тесты готовы. Аллюр готов. PageObject'ы на три разных страницы готовый.

nikolay-977 commented on src/test/java/ChromeTest/AuthorizationChromeTest.java:
```diff
@@ -0,0 +1,112 @@
+        LoginPageObject.fieldPassword.setValue(userPassword);
+        LoginPageObject.buttonEnterOrRegistration.click();
+        MainPageObject.orderThat.shouldHave(Condition.exactText("Оформить заказ"));
+        closeWebDriver();
```
Вызов closeWebDriver() лучше вынести в отдельный метода с аннотацией @After, иначе в случае фейла на 
 проверке MainPageObject.orderThat.shouldHave(Condition.exactText("Оформить заказ")) не будет вызван, что может привести к фейлу других тестов.

nikolay-977 commented on src/test/java/YandexBrowserTest/RegistrationYandexTest.java:
```diff
@@ -0,0 +1,85 @@
+        }
+
+    @Step
+    public void tearDown(){
```
Здесь нужна аннотация @After.

apologet-insane committed:
Исправления:
1. Критериий "Все элементы страниц, которые ты используешь в тестах, описаны в Page Object." подразумевает, что нужно указать в PageObject все элементы, используемые в тестах. Прикрываюсь мнением наставника группы Sun, уточнил у него трактовкку критерия.

2. От всех div избавиться не смог. Но по максимуму заменил их на что-то другое.
3. Перенёс в PageObject методы и прописал каждому @Step. Крутая штука, не знал, что так можно.
4. Переделал проверки на assert, добавил им сообщения.
5. Убрал closeWebdriver() в @After

apologet-insane committed:
1. Раздели тест регистрации на позитивный и негативный
2. Поместил open внутрь тестов в Chrome.
3. Вынес везде closeWebDriver в @After
4. Добавил полноэкранный режим
